x-pack/plugin/otel-data/src/main/resources/component-templates/semconv-resource-to-ecs@mappings.yaml

```
@@ -0,0 +1,133 @@
+---
+version: ${xpack.oteldata.template.version}
+_meta:
+  description: Aliases from OpenTelemetry SemConv fields to ECS (and some non-ECS) fields
+  managed: true
+template:
+  mappings:
+    properties:
+      resource:
+        properties:
+          attributes:
+            type: passthrough
+            dynamic: true
+            priority: 30
+            time_series_dimension: true
+            properties:
+              telemetry.sdk.language:
+                type: keyword
+                ignore_above: 1024
+              service.instance.id:
+                type: keyword
+                ignore_above: 1024
+              deployment.environment:
+                type: keyword
+                ignore_above: 1024
+              cloud.platform:
+                type: keyword
+                ignore_above: 1024
+              container.image.tags:
+                type: keyword
+                ignore_above: 1024
+              host.arch:
+                type: keyword
+                ignore_above: 1024
+              process.executable.path:
+                type: keyword
+                ignore_above: 1024
+              process.runtime.path:
```

```suggestion
              process.runtime.name:
```
I think this should be `process.runtime.name`